optaplanner-core/src/test/java/org/optaplanner/core/impl/solver/DefaultSolverTest.java

```diff
@@ -111,4 +115,25 @@ public void solveStopsWhenPartiallyInitialized() {
         assertSame(solution, solver.getBestSolution());
     }
 
+    @Test(timeout = 600_000)
+    public void zeroEntity() {
+        SolverFactory<TestdataChainedSolution> solverFactory = PlannerTestUtils.buildSolverFactory(
+                TestdataChainedSolution.class, TestdataChainedEntity.class);
+
+        LocalSearchPhaseConfig phaseConfig = new LocalSearchPhaseConfig();
+        phaseConfig.setTerminationConfig(new TerminationConfig().withStepCountLimit(1));
+        solverFactory.getSolverConfig().setPhaseConfigList(Collections.singletonList(phaseConfig));
+
+        Solver<TestdataChainedSolution> solver = solverFactory.buildSolver();
+
+        TestdataChainedSolution solution = new TestdataChainedSolution("1");
```

Comment:
The domain configuration is a bit different in the original test

Reply:
Is that a problem?